jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/TableParameterAligner.java

```diff
@@ -7,6 +7,11 @@
 import java.util.List;
 
 public class TableParameterAligner {
+
+    private TableParameterAligner() {
```

Comment:
Nitpick, but I would probably use `@UtilityClass` since we already use it in other places.